RELEASE_PROCESS.md

```diff
@@ -49,30 +49,37 @@ Starting download https://packages.chef.io/files/current/chef-server/14.2.23/ubu
 
 ### Testing the Release
 
-Every merge to chef-server master must be built, and this build must be tested with the full Umbrella automated integration test pipeline at https://buildkite.com/chef/chef-umbrella-master-chef-server-full.
+Every merge to chef-server master must be built, and this build must be tested with the full Umbrella automated integration test pipeline at https://buildkite.com/chef/chef-umbrella-master-chef-server-full (NOTE: use https://buildkite.com/chef/chef-umbrella-master-chef-server for testing the 12.17.15 upgrade path).
```

Comment:
Every merge to master automatically runs through adhoc and we run the umbrella nightly for all the changes made in that day.

Reply:
I added this in the form of a NOTE.